publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/controller/ProducerController.java

```diff
@@ -145,39 +149,48 @@ public ResponseEntity generateAndPublish(@ApiParam(value = "message protocol", r
                                              @ApiParam(value = "JSON message", required = true) @RequestBody final JsonObject bodyJson) {
 
         URLTemplate urlTemplate = new URLTemplate();
```

Comment:
Can we rename it to something internal like ex: GenerateURLTemplate, URLTemplate looks very Generic.